packages/rest/src/validation/request-body.validator.ts

```diff
@@ -142,13 +142,18 @@ function validateValueAgainstSchema(
     cache.set(schema, validatorMap);
   }
 
-  if (validate(body)) {
-    debug('Request body passed AJV validation.');
-    return;
+  let validationErrors: AJV.ErrorObject[] = [];
+  try {
+    const validationResult = await validate(body);
+    // When schema is not available, ajv returns null
+    if (validationResult || validationResult === null) {
+      debug('Request body passed AJV validation.');
+      return;
+    }
```

Comment:
Seems like this branch isn't fully covered with tests yet (hence the reduced test coverage).

Is there a case where the schema isn't available when AJV is called?

It'll be great to add a test case to prevent regression.

Here's the [Coveralls report](https://coveralls.io/builds/29048569/source?filename=packages%2Frest%2Fsrc%2Fvalidation%2Frequest-body.validator.ts#L149)

Reply:
Hi @achrinza, [this](https://github.com/strongloop/loopback-next/pull/4762/files#diff-abbe6f4cfbede11c53ba54c636a7b24dR209-R214) test case doesn't pass without `validationResult === null` as promise returned by `validate` returns null.

My comment there seems misleading with `schema is not available`. Schema is available but body is optional and value is empty.

Reply:
@achrinza did you get a chance to look into this?

Reply:
@mastermunj I would suggest editing the comment to be more accurate to the purpose of the conditional statement.

Though @raymondfeng has approved the PR and this issue is a non-blocker for me.

Reply:
> My comment there seems misleading with `schema is not available`. Schema is available but body is optional and value is empty.

I would really like the commit to be updated before this pull request is landed, to avoid confusion in the future.